pkg/apis/flipop/v1alpha1/flipop_types.go

```diff
@@ -161,7 +173,8 @@ type NodeDNSRecordSetList struct {
 
 // NodeDNSRecordSetStatus defines the observed state of NodeDNSRecordSet.
 type NodeDNSRecordSetStatus struct {
-	Error string `json:"error,omitempty"`
+	Error string             `json:"error,omitempty"`
+	State NodeDNSRecordState `json:"state"`
```

Comment:
Making this change will require regenerating the client set.  `make generate-k8s`